src/files/BrsFile.ts

```diff
@@ -224,13 +212,9 @@ export class BrsFile {
     public parse(fileContents: string) {
         try {
             this.fileContents = fileContents;
-            if (this.parseDeferred.isCompleted) {
```

Comment:
Deleting this condition no longer guards against `parse` being called more than once. I'm ok with that, but then we need to ensure that any setup/initialization steps are replicated in the `parse` method (like clearing out the `diagnostics` array as one example of many).